### DISCUSSION TITLE
Sem Pressão

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#402] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/sem-pressao/

**Comment by brunoaugusto** [comment]
Essa prova é sacanagem. Não sei como é hoje, mas eu paguei R$ 80 pra fazer, mais da metade das perguntas eu sequer aprendi no ensino médio e pra poder levar minha cópia das perguntas embora tive que esperar todo o tempo do teste.
